imap_processing/tests/test_decom.py

```diff
@@ -0,0 +1,61 @@
+"""Tests aspects of the decommutation process that is generic to all instruments"""
+
+import glob
+
+import pytest
+
+from imap_processing import imap_module_directory
+
+xtce_documents = glob.glob(f"{imap_module_directory}/*/packet_definitions/*.xml")
+xtce_documents = [
+    item for item in xtce_documents if "idex_packet_definition.xml" not in item
+]  # Temporary until IDEX packet definition is fixed
+
+
+@pytest.mark.parametrize("xtce_document", xtce_documents)
+def test_ccsds_header(xtce_document):
+    """Test if the XTCE document contains the proper CCSDS header information"""
+
+    print(f"\nTesting {xtce_document}")
+
+    # Define what is expected in the XTCE document
```

Comment:
Reference here for what space packet parser is checking for I believe.
https://github.com/medley56/space_packet_parser/blob/bef50e6bfd262fdef4727381c5dafea6488571c2/space_packet_parser/parser.py#L20-L27